packages/amplify_core_plugin_interface/lib/amplify_datastore_category.dart

```diff
@@ -47,4 +47,10 @@ class DataStoreCategory {
             where: where, pagination: pagination, sortBy: sortBy)
         : throw (errorMsg);
   }
+
+  Future<T> delete<T extends Model>(T model, {QueryPredicate when}) {
+    return plugins.length == 1
+        ? plugins[0].delete(model, when: when)
+        : throw (errorMsg);

```

Comment:
Do we need the parenthesis?

Reply:
I'm actually not sure, but this is what we do in the other APIs.